collect_app/src/androidTest/java/org/odk/collect/android/espressoutils/pages/Page.java

```diff
@@ -161,4 +168,35 @@ public T checkIfAreaWithKeyIsDisplayed(String key) {
         onData(PreferenceMatchers.withKey(key)).check(matches(isDisplayed()));
         return (T) this;
     }
+
+    public T checkIfStringIsEnabled(int stringID) {
+        onView(withText(getInstrumentation().getTargetContext().getString(stringID))).check(matches(not(isEnabled())));
+        return (T) this;
+    }
+
+    public T scrollToElementWithKey(String key) {
```

Comment:
All those methods that use `PreferenceMatchers` like this one are related to settings, so it would be good to have one parent class called eg. `SettingsPage` where we could move them. Such a class should be extended by classes like `AdminSettingsPage`, `GeneralSettingsPage`, `ServerSettingsPage`, `UserAndDeviceIdentitySettingsPage` then.

Reply:
I think that it would be better to add it in another pr as it is not specifically connected with added tests.

Reply:
Ok.